test/solution-template-validation-tests/test/mainTemplateTests.js

```diff
@@ -118,6 +119,7 @@ describe('template files - ', () => {
             /** resourceGroup().location should NOT be present anywhere in template, EXCEPT as a defaultValue */
             it.each(templateObject, 'resourceGroup().location must NOT be be used in the template file ' + templateJSONObject.filename + ', except as a default value for the location parameter. Please correct similar errors in the file', function(element, next) {
                 var templateFileContent = JSON.stringify(templateObject);
```

Comment:
Please check if there is a "minify" json method that removes spaces/carriage return, so we wouldn't have to check for space/special chars in our regexes

Reply:
Unfortunately, the stringify is the only json method to transfer json to string. If we would like a method that removes spaces/carriage return, we need to implement one in util.js.